nativescript/README.md

```diff
@@ -0,0 +1,52 @@
+#NativeScript implementation with Angular 2
+
+Visit the [NativeScript page](http://propertycross.com/nativescript/) on the PropertyCross website for screenshots.
+
+##Introduction
+
+[NativeScript](https://www.nativescript.org/) is a framework to develop truly native apps for iOS and Android from a single code base of JavaScript or TypeScript, XML and CSS.
+NativeScript takes the cross-platform code and translates it into the language that the target platform speaks.
```

Comment:
What does this actually mean? Is it transpiled into ObjC / Java then compiled for the specific platform?